connPFM/connectivity/connectivity_utils.py

```diff
@@ -164,7 +169,7 @@ def sum_histograms(
     all_hists = np.zeros((len(hist_list), hist_list[0][3].shape[0] - 1))
 
     for rand_idx in range(len(hist_list)):
-        all_hists[rand_idx, :] = hist_list[rand_idx][2]
+        all_hists[rand_idx, :] = hist_list[rand_idx][1]
```

Comment:
Why has this changed? I haven't worked with the code in a long time so I'm probably missing something.

Reply:
Because I just changed the return for the rss_sur function, so now the histogram is the second ouptut not the third